Makefile

```diff
@@ -0,0 +1,104 @@
+# Makefile for packaging and testing this app. Should follow make contract
```

Comment:
I think this makefile is mostly superfluous. The tests pass fine just running `tox`. The only benefits of the makefile AFAICT are:
- Combining coverage information between all the different tox environments.
- A target for tagging the releases.
- It functions in environments which don't have tox already installed.

WDYT about just getting rid of it? If we do want additional boilerplate for running the test suite, it would ideally be a `.travis.yml` file.